XPlat.Devices.Geolocation/Geolocator.Android.cs

```diff
@@ -38,7 +45,7 @@ public class Geolocator : IGeolocator
         /// </param>
         public Geolocator(Context context)
         {
-            this.locationManager = (LocationManager)context.GetSystemService(Context.LocationService);
+            this.locationManager = (LocationManager)Android.App.Application.Context.GetSystemService(Context.LocationService);
```

Comment:
Looks like when you've re-added the constructor parameter for the context, this has not been reverted back to the previous change.